igvc_navigation/launch/octomap.yaml

```diff
@@ -0,0 +1,76 @@
+octree:
+    resolution: 0.2 # width & length of each grid cell in m
+    clamping:
+        max: 0.95 # Value of octree considered 100% occupied stable
+        min: 0.05 # Value of octree considered 0% occupied stable
+# P(x | x) where x is either miss or hit. So prob_miss: 0.2 means that I'm 20% confident a miss is a miss
+probability_model:
+    lidar:
+        scan:
+            prob_miss: 0.55
+            prob_hit: 0.9
+        ground:
+            prob_miss: 0.6
+            prob_hit: 0.6 # Not used
+        free_space:
+            prob_miss: 0.52
+            prob_hit: 0.6 # Not used
+    camera:
+        prob_miss: 0.45
+        prob_hit: 0.9
+sensor_model:
+    max_range: 5
+    angular_resolution: 0.01
+filters:
+    behind:
+        enabled: true
+        filter_angle: 2
+        distance: 3
+    empty:
+        enabled: true
+        start_angle: -0.5 #Radians
+        end_angle: 0.5
+        miss_cast_distance: 5
+        max_range: 5
+    empty_image:
+        blur:
+            kernel: 17
+            sigma: 11
+        threshold: 0.5
+    combined_map:
+        blur:
```

Comment:
add a brief comment describing how modifying these parameters will change the resulting map